config.json

```diff
@@ -175,7 +175,7 @@
           "path": "Tools/msbuild.cmd"
         },
         "unix":{
-          "defaultParameters": "/nologo /verbosity:minimal /clp:Summary /maxcpucount /l:BinClashLogger,Tools/Microsoft.DotNet.Build.Tasks.dll;LogFile=binclash.log",
+          "defaultParameters": "/nologo /verbosity:minimal /clp:Summary /maxcpucount /l:BinClashLogger,Tools/Microsoft.DotNet.Build.Tasks.dll;LogFile=binclash.log /p:SkipTests=true",
```

Comment:
I think I would prefer it to be somewhere we can add a comment about it, and file an issue on it because we should be able to run tests on unix as well.